server/src/storage/compressed_data_manager.rs

```diff
@@ -335,8 +335,8 @@ async fn is_object_meta_relevant(
 /// Return [`true`] if the timestamps and values in `file name` overlaps with the time range given
 /// by `start_time` and `end_time` and the value range given by `min_value` and `max_value`,
 /// otherwise [`false`]. Assumes `file_name` has the following format:
-/// `start_timestamp-end_timestamp-min_value-max_value.parquet`, where both `start_timestamp` and
-/// `end_timestamp` are of the same unit as `start_time` and `end_time`.
+/// `start-timestamp_end-timestamp_min-value_max_value.parquet`, where both `start-timestamp` and
```

Comment:
Very minor but it seems that `max_value` was not changed to `max-value`.

Reply:
Fixed.